JetStreamCompose/jetstream/build.gradle

```diff
@@ -80,20 +90,24 @@ kapt {
 dependencies {
     def tv_compose_version = "1.0.0-SNAPSHOT"
 //    def tv_compose_version = "1.0.0-alpha06"
-    def core_ktx_version = "1.12.0-alpha04"
+    def core_ktx_version = "1.10.1"
```

Comment:
Why was this downgraded?

Reply:
It was throwing some errors after upgrading AGP to version `8.0.2`. 
I found that the recommended fix was to switch to the latest stable release.